test/core/security/oauth2_utils.cc

```diff
@@ -110,6 +108,9 @@ char* grpc_test_fetch_oauth2_token_with_credentials(
   gpr_mu_unlock(request.mu);
 
   grpc_pollset_shutdown(grpc_polling_entity_pollset(&request.pops),
-                        &destroy_after_shutdown_closure);
+                        &do_nothing_closure);
```

Comment:
Why define do_nothing and do_nothing_closure? You can pass nullptr rather than an empty closure.

```
    grpc_pollset_shutdown(grpc_polling_entity_pollset(&request.pops), nullptr);
```

@vjpai may have better suggestion.

Reply:
Just followed some existing pattern but it looks like the nullptr approach also works (just tested it).  @vjpai can you please confirm that it's safe.

Reply:
Looks like there are more uses of a do-nothing closure but it also looks like nullptr is acceptable. I'm guessing that maybe we switched our expectations at some point